src/api/v1/endpoint/chat.py

```
@@ -1,17 +1,85 @@
+    try:
+        result_data = await generate_answer(
+            query=request.query, db=db, country_id=request.country_id
+        )
+
+        chat_result = ChatResult(**result_data)
+
+        return CommonResponse(
+            isSuccess=True, code="AI200", message="성공입니다.", result=chat_result
+        )
+
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)